deployments/add-ons/prometheus/README.md

```diff
@@ -0,0 +1,27 @@
+## Steps to setup Prometheus and Amazon Managed Service for Prometheus
+1. Create an IAM Policy:
+    1. `aws iam create-policy —policy-name AMPIngestPolicy —policy-document file://deployments/add-ons/prometheus/AMPIngestPermissionPolicy.json`
+    2. You will need to copy the ARN, as you will need to use it in the next command replacing <your-policy-arn>
+2. Create a Service Account:
+    1. `eksctl create iamserviceaccount --name amp-iamproxy-ingest-service-account --role-name <your-role-name> --namespace monitoring --cluster \
+<your-cluster-name> --attach-policy-arn <your-policy-arn> --override-existing-serviceaccounts --approve --region <your-cluster-region>`
```

Comment:
Why is role-name required? Doesn't `eksctl` create a random role when given a policy arn?

Also the env variables from above can be substituted here:
```
eksctl create iamserviceaccount --name amp-iamproxy-ingest-service-account --namespace monitoring --cluster \
$CLUSTER_NAME --attach-policy-arn $AMP_POLICY_ARN --override-existing-serviceaccounts --approve --region $CLUSTER_REGION
```